src/Raven.Server/Commercial/LicenseManager.cs

```diff
@@ -703,7 +704,7 @@ private async Task ExecuteTasks()
             }
         }
 
-        public async Task LeaseLicense(bool forceUpdate = false)
+        public async Task LeaseLicense(string guid, bool forceUpdate = false)
```

Comment:
```suggestion
        public async Task LeaseLicense(string raftReqId, bool forceUpdate = false)
```

Or something like that